backend/app/services/adapters/team_kinds.py

```
@@ -2142,5 +2142,172 @@ def get_team_skills(
+        Non-solo mode: shallow copy — new team references the same bots.
+        """
+        from app.schemas.team import BotInfo, TeamCreate
+        from app.services.adapters.bot_kinds import bot_kinds_service
+
+        # Fetch the original team
+        original = (
+            db.query(Kind)
+            .filter(
+                Kind.id == team_id,
+                Kind.kind == "Team",
+                Kind.is_active == True,
```

Ruff (0.15.12)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)